### DISCUSSION TITLE
Communication with subprocess not working in github actions

### DISCUSSION THREAD

**Discussion by p5k369** [eoyilmaz/displaycal-py3#74] [General]
Hey @eoyilmaz,
what do you think about mocking the call to communicate of Popen? As far as I can tell everything works as expected. Locally the test test_get_argyll_version_string_1 works as expected. On github actions, for whatever reason, we are able to start the process but after that get no data from STDOUT or STDERR and because no timeout was specified with communicate we are listening forever.

Would you accept a pull request that mocks Popen.communicate ?

**Comment by eoyilmaz** [comment]
Hey @p5k369, yeah sure, go for it :+1:
